python/cuml/test/test_kmeans.py

```diff
@@ -70,8 +68,14 @@ def test_kmeans_sklearn_comparison(name, nrows):
             assert (calculation < 2e-3) and score_test
 
         else:
+            if name == 'aniso':
```

Comment:
should we omit or xfail this dataset for now if it's really significantly off from the baseline rather than adapting the tolerance?

Reply:
it's not super off really, I think 15 points in the frontier of the clusters, so I think it still is nice to have it here to detect any potentially bigger change in behavior

Reply:
I can live with that